src/main/java/com/hazelcast/remotecontroller/CloudManager.java

```diff
@@ -163,13 +163,12 @@ public CloudCluster getCloudCluster(String clusterId, boolean setupTls) throws C
                 }
                 String responseString = responseBody.string();
                 throwIfResponseFailed(res, responseString);
-                // LOG.info(maskValueOfToken(responseString));
                 JsonNode rootNode = mapper.readTree(responseString);
                 if (rootNode.asText().equalsIgnoreCase("null")) {
                     throw new CloudException(String.format("Cluster with id %s is not found", clusterId));
                 }
                 boolean tlsEnabled = rootNode.get("tlsEnabled").asBoolean();
-                CloudCluster cluster = new CloudCluster(rootNode.get("id").asText(), rootNode.get("name").asText(), rootNode.get("releaseName").asText(), rootNode.get("hazelcastVersion").asText(), tlsEnabled, rootNode.get("state").asText(), rootNode.get("tokens").elements().next().get("token").asText(), null, null);
+                CloudCluster cluster = new CloudCluster(rootNode.get("id").asText(), rootNode.get("name").asText(), rootNode.get("name").asText(), rootNode.get("hazelcastVersion").asText(), tlsEnabled, rootNode.get("state").asText(), rootNode.get("tokens").elements().next().get("token").asText(), null, null);
```

Comment:
did the rest api response change for Viridian? It was expecting a releaseName before int he response. Can you point me to the documentation for this REST API? If this field is removed then it is better to remove it from CloudCluster constructer as well.